.github/workflows/code-coverage.yml

```diff
@@ -5,7 +5,7 @@
 
 name: Code Coverage Report
 
-on: [push, pull_request]
+on: [push]
```

Comment:
I think we want the code coverage to run on pull requests as well, so the developer knows if their code coverage meets the guideline before they merge to develop

Reply:
That makes sense, I'll put that back in.